packages/trader/src/Modules/MT5/Components/mt5-poa.jsx

```diff
@@ -0,0 +1,453 @@
+import { Formik }            from 'formik';
+import PropTypes             from 'prop-types';
+import React, { Component }  from 'react';
+import {
+    FormSubmitButton,
+    ThemedScrollbars,
+    Dropdown,
+    Loading,
+}                            from '@deriv/components';
+import {
+    Localize,
+    localize,
+}                            from '@deriv/translations';
+import { WS }                from 'Services/ws-methods';
+import { FormSubHeader }     from 'Modules/Account/Components/layout-components.jsx';
+import FileUploaderContainer from 'Modules/Account/Sections/Verification/ProofOfAddress/file-uploader-container.jsx';
+import { poa_status_codes }  from 'Modules/Account/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx';
+import {
+    Expired,
+    NeedsReview,
+    Unverified,
+    Verified,
+    Submitted,
+}                            from 'Modules/Account/Sections/Verification/ProofOfAddress/proof-of-address-messages.jsx';
+import {
+    validAddress,
+    validLength,
+    validPostCode,
+}                            from 'Utils/Validator/declarative-validation-rules';
+import { InputField }        from './mt5-personal-details-form.jsx';
+
+const form = React.createRef();
```

Comment:
What is this for? I couldn't find any references to its usage (apart from passing it to `<Formik />`).